### PR TITLE
copy update: contact us page

### PR DESCRIPTION
**Pull request by pnavarrc**
[Trello 351](https://trello.com/c/PPGnj6mi/351-implement-copy-changes-in-web-copy-contact-us) - Update the copy on the _Contact us_ page.

- [Web Copy // Contact us](https://docs.google.com/document/d/1YlX4zY9O5WeRvYg54Cc-pPRg7v9ZNADQjEP_-59WD4o/edit?ts=5f28a37c&pli=1#)

## Notes
- The JD for the internship positions might not be finalized, I will get confirmation before merging, but that is not blocking to review.

### CODE REVIEW

**Comment by pnavarrc** [comment]
[Implement copy changes in Web Copy // Contact us](https://trello.com/c/PPGnj6mi/351-implement-copy-changes-in-web-copy-contact-us)

File: src/screens/Contact/Contact.tsx [hunk 1]
```diff
@@ -35,123 +48,122 @@ const Contact = ({ children }: { children: React.ReactNode }) => {
...
+            . We're currently offering full-time internships (volunteer):
+          </Paragraph>
+          <ul>
+            <ListItem>
+              <ExternalLink href="https://docs.google.com/document/d/1_4P0lM1k2pGRXSNGVe_p8TcmdMdaIug5lZeIK-3zU24/edit?ts=5f2488d3#">
+                Strategy & Operations
+              </ExternalLink>
+            </ListItem>
+            <ListItem>
+              <ExternalLink href="https://docs.google.com/document/d/1HJJ8-W87Z1dGVK1JZF3cDNVyBHSq-Z1MYVoVL4ofZj0/edit#">
```
Comment (mikelehen) [inline src/screens/Contact/Contact.tsx:131]: This doc isn't public.

Comment (pnavarrc) [reply src/screens/Contact/Contact.tsx:131]: good catch - will get this fixed before merging 👍

Comment (pnavarrc) [reply src/screens/Contact/Contact.tsx:131]: Debbie updated the links, merging!

**Pull request merged**